source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/IntegrationTests/WebApi/QueryServices/ChargesQueryServiceTests.cs

```diff
@@ -0,0 +1,447 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Energinet.Charges.Contracts.Charge;
+using FluentAssertions;
+using GreenEnergyHub.Charges.Domain.Dtos.SharedDtos;
+using GreenEnergyHub.Charges.Infrastructure.Persistence;
+using GreenEnergyHub.Charges.IntegrationTest.Core.Fixtures.Database;
+using GreenEnergyHub.Charges.QueryApi;
+using GreenEnergyHub.Charges.QueryApi.Model;
+using GreenEnergyHub.Charges.QueryApi.QueryServices;
+using GreenEnergyHub.Charges.TestCore;
+using GreenEnergyHub.Charges.TestCore.Builders.Command;
+using Microsoft.EntityFrameworkCore;
+using NodaTime.Extensions;
+using Xunit;
+using Xunit.Categories;
+using Charge = GreenEnergyHub.Charges.Domain.Charges.Charge;
+using ChargePeriod = GreenEnergyHub.Charges.Domain.Charges.ChargePeriod;
+using MarketParticipant = GreenEnergyHub.Charges.Domain.MarketParticipants.MarketParticipant;
+
+namespace GreenEnergyHub.Charges.IntegrationTests.IntegrationTests.WebApi.QueryServices
+{
+    [IntegrationTest]
+    public class ChargesQueryServiceTests : IClassFixture<ChargesDatabaseFixture>
+    {
+        private const string MarketParticipantOwnerId = "MarketParticipantId";
+
+        // Is being set when executing the SeedDatabase method
```

Comment:
This comment is already outdated, so It should either be as suggested below or maybe just removed.
```suggestion
        // Is being set when executing the `GetOrAddMarketParticipantAsync` method
```

Reply:
Removed it.